static/src/javascripts/projects/common/utils/detect.js

```diff
@@ -133,6 +133,14 @@ define([
         return navigator.mozApps && !window.locationbar.visible;
     }
 
+    function isFacebookApp() {
+        return navigator.userAgent.indexOf('FBAN/') > -1;
+    }
+
+    function isTwitterApp() {
+        return navigator.userAgent.indexOf('Twitter for iPhone') > -1;
```

Comment:
Adding a comment to that effect will help future users reading your code :dancer: